.github/workflows/build.yml

```diff
@@ -44,6 +40,9 @@ jobs:
           TrueLayer__ClientId: ${{ secrets.TRUELAYER__CLIENTID }}
           TrueLayer__ClientSecret: ${{ secrets.TRUELAYER__CLIENTSECRET }}
           TrueLayer__Payments__SigningKey__KeyId: ${{ secrets.TRUELAYER__PAYMENTS__SIGNINGKEY__KEYID }}
+          TrueLayer2__ClientId: ${{ secrets.TRUELAYER__CLIENTID }}
```

Comment:
Are these required by cake?

Reply:
it's to generate the appsettings with two clients

Reply:
Yeah i've noticed only after that they are used from one of the acceptance tests. ignore this